tools/wrangling/.shed.yml

```diff
@@ -0,0 +1,8 @@
+categories:
+  - Text Manipulation
+description: Convert an xlsx file to a tabular
+long_description: Extract one sheet from an xlsx/xls file and convert it to a tabular format
+name: excel_to_tabuler
+owner: ufz
+homepage_url: https://github.com/bernt-matthias/mb-galaxy-tools
```

Comment:
Can you update the URLs to `Helmholtz-UFZ` instead of `bernt-matthias`. 

`remote_repository_url` is wrong.